bin/docs

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+# Click requires us to ensure we have a well configured environment to run
+# our click commands. So we'll set our environment to ensure our locale is
+# correct.
+export LC_ALL="${ENCODING:-en_US.UTF-8}"
+export LANG="${ENCODING:-en_US.UTF-8}"
+
+COMMAND_ARGS=$@
```

Comment:
we're not using this, or did I miss something ? (also, even if this isn't actually relevant, usually, a good rule of thumb is that`"$@"` is most often what we mean when we use `$@`. Missing the double quotes is a bear trap)

Reply:
Just some leftover copypasta, I removed it.